docs/handbook/performance/inserts/tuning.rst

```diff
@@ -98,7 +98,22 @@ Translog
 If `translog.durability`_ is set to ``REQUEST`` (the default), the translog
 gets flushed after every operation. Setting this to ``ASYNC`` will improve
 insert performance, but it also worsens durability. If a node crashes before a
-translog has been synced, those opperations will be lost.
+translog has been synced, those operations will be lost.
+
+Overload Protection
+-------------------
+
+The :ref:`Overload Protection <crate-reference:overload_protection>` settings
```

Comment:
@ckurze: This shows how you can directly link to another documentation section which is also based on Sphinx.